Sources/Orbit/Components/Button.swift

```diff
@@ -25,43 +25,52 @@ public struct Button: View {
                 action()
             },
             label: {
-                HStack(spacing: .xSmall) {
+                HStack(spacing: 0) {
                     if disclosureIconContent.isEmpty {
                         Spacer(minLength: 0)
                     }
 
-                    Icon(iconContent, size: iconSize)
-
-                    if #available(iOS 14.0, *) {
-                        Text(
-                            label,
-                            size: size.textSize,
-                            color: .custom(style.foregroundUIColor),
-                            weight: .medium,
-                            linkColor: .custom(style.foregroundUIColor)
-                        )
-                    } else {
-                        Text(
-                            label,
-                            size: size.textSize,
-                            color: .custom(style.foregroundUIColor),
-                            weight: .medium,
-                            linkColor: .custom(style.foregroundUIColor)
-                        )
-                        // Prevents text value animation issue due to different iOS13 behavior
-                        .animation(nil)
+                    HStack(alignment: .firstTextBaseline, spacing: .xSmall) {
+                        Icon(iconContent, size: iconSize)
+                        text
+                            .padding(.vertical, size.verticalPadding)
                     }
 
                     Spacer(minLength: 0)
 
+                    TextStrut(size.textSize)
```

Comment:
When there is no text, only an icon - or any combination of them, we need to have a single fixed height when such buttons are next to each other (44pt in case of default font size). The padding around the text itself is needed too, in case of multiline text. So we need both paddings - one to enforce the minimal size (based on text size) and the other to create padding around the content.

Reply:
Is it possible to have a multiline label on the button? Previews for some reason don't work

Reply:
It is possible, yes, we will need to support it with bigger font sizes in future.

Previews do not work by opening the package?

Reply:
I 💯 sure it's something local, there is no error just out of time error. Will try with clean build